tests/contrib/fastapi/test_fastapi_patch.py

```diff
@@ -0,0 +1,19 @@
+from ddtrace.contrib.fastapi import patch
+from ddtrace.contrib.fastapi import unpatch
+from tests.contrib.patch import PatchTestCase
+
+
+class TestFastapiPatch(PatchTestCase.Base):
+    __integration_name__ = "fastapi"
+    __module_name__ = "fastapi"
+    __patch_func__ = patch
+    __unpatch_func__ = unpatch
+
+    def assert_module_patched(self, fastapi):
+        self.assert_wrapped(fastapi.applications.FastAPI.__init__)
```

Comment:
We are missing testing for `serialize_response`.

https://github.com/DataDog/dd-trace-py/blob/1595f9f9d/ddtrace/contrib/fastapi/patch.py#L72-L72